videochat-conf/src/config.js

```diff
@@ -21,45 +21,49 @@ export const messages = {
 export const E2E_STATE_ELEMENT_ID = 'e2e-ice-state';
 
 export const credentials = {
-  appId: 385,
-  authKey: "DFBMs5-dKBBCXcd",
-  authSecret: "SkCW-ThdnmRg9Za"
+  appId: REPLACE_APP_ID,
+  authKey: "REPLACE_APP_AUTH_KEY",
+  authSecret: "REPLACE_APP_AUTH_SECRET"
 };
 
 export const appConfig = {
   debug: { mode: 1 },
-  conference: { server: 'wss://janus.connectycube.com:8989' },
+  conference: { server: 'wss://REPLACE_APP_JANUS_DOMAIN:8989' },
+  endpoints: {
```

Comment:
for a sake of simplicity, let's remove endpoints configuration code
Shared users do not need this to change